### PR TITLE
Add support for rbac to fiaas-skipper helm chart

### PR DESCRIPTION
**Pull request by birgirst**

### CODE REVIEW

File: helm/fiaas-skipper/templates/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{{- if .Values.rbac.enabled }}
+kind: ClusterRole
+{{- if semverCompare "^1.8-0" .Capabilities.KubeVersion.GitVersion }}
```
Comment (mortenlj) [inline helm/fiaas-skipper/templates/rbac.yaml]: Which versions does this match? I can't find any documentation on what happens when the version after the hyphen is `0`. Would it be better to just use `">=1.8"`?

File: helm/fiaas-skipper/templates/rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{{- end }}
+metadata:
+  name: fiaas-controller
+rules:
+  - apiGroups:
+      - fiaas.schibsted.io
+    resources:
+      - applications
+      - application-statuses
+      - paasbetaapplications
```
Comment (mortenlj) [inline helm/fiaas-skipper/templates/rbac.yaml]: `paasbetaapplications` and `paasbetastatuses` are in the `schibsted.io` group, not `fiaas.schibsted.io`.

Comment (birgirst) [reply helm/fiaas-skipper/templates/rbac.yaml]: Fixed.

File: helm/fiaas-skipper/templates/rbac.yaml (hunk 1 above)
Comment (birgirst) [reply helm/fiaas-skipper/templates/rbac.yaml]: I've updated to use >=1.8

**Pull request merged**